modules/jupyter-widget/src/widget.js

```diff
@@ -0,0 +1,112 @@
+import {DOMWidgetModel, DOMWidgetView} from '@jupyter-widgets/base';
+
+import {MODULE_NAME, MODULE_VERSION} from './version';
+
+import {createDeckScaffold, loadCss, hideMapboxCSSWarning, setMapProps} from './utils';
+
+const mapboxgl = require('mapbox-gl');
+const deckgl = require('@deck.gl/core');
+const deckglLayers = require('@deck.gl/layers');
+const deckAggregationLayers = require('@deck.gl/aggregation-layers');
+const deckJson = require('@deck.gl/json');
+
+const MAPBOX_CSS_URL = 'https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.1/mapbox-gl.css';
+const MAPBOX_TILE_URL = 'mapbox://styles/mapbox/light-v9';
+
+export class DeckGLModel extends DOMWidgetModel {
+  defaults() {
+    return {
+      ...super.defaults(),
+      _model_name: DeckGLModel.model_name,
+      _model_module: DeckGLModel.model_module,
+      _model_module_version: DeckGLModel.model_module_version,
+      _view_name: DeckGLModel.view_name,
+      _view_module: DeckGLModel.view_module,
+      _view_module_version: DeckGLModel.view_module_version,
+      json_input: null
+    };
+  }
+
+  static get serializers() {
+    return {...DOMWidgetModel.serializers};
+    // Add any extra serializers here
+  }
+
+  static get model_name() {
```

Comment:
Are you only using these in the defaults above? In that case maybe just define them as constants at the top of the file?